build.gradle

```diff
@@ -204,20 +209,25 @@ for ( sv in ['2_10_5', '2_11_7'] ) {
   }
 }
 
-tasks.create(name: "jarAll", dependsOn: ['jar_core_2_10_5', 'jar_core_2_11_7', 'clients:jar', 'examples:jar', 'contrib:hadoop-consumer:jar', 'contrib:hadoop-producer:jar', 'log4j-appender:jar', 'tools:jar']) {
+def copycatPkgs = ['copycat-data', 'copycat-api', 'copycat-runtime', 'copycat-json', 'copycat-file']
```

Comment:
data + api can definitely be combined. I personally like using this separation to enforce clean layering, but there's not much risk of messing that up with those two.

The reason for splitting api and runtime is that connector developers should only need the api jar. Again, this is a way to really force proper layering, be clear about what we want to expose as public API to connector developers, etc.